public/_locales/en/messages.json

```diff
@@ -1699,5 +1699,11 @@
   },
   "blockExplorer": {
     "message": "Blockchain explorer"
+  },
+  "copiedAmount": {
+    "message": "Copied."
+  },
+  "copyAmountToClipboard": {
+    "message": "Copy to clipboard"
```

Comment:
These values already exist in message. Use them pls.

Reply:
I haven't done this way because grammars of other languages may bring you lots of surprises, especially when they have genders and aren't Indo-European.